Dockerfile.debian

```diff
@@ -0,0 +1,73 @@
+# syntax = docker/dockerfile:1.0-experimental
```

Comment:
This is mostly a copy of our k8s image dockerfile but with the 2nd half ported to debian for ease of use during testing.

Docker doesn't seem to have a mechanism for #include'ing other dockerfiles so I just copy and pasted the first half to avoid having to rewrite large sections of the Makefile